streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -789,7 +789,7 @@ private void prepareChangelogs(final Set<ChangelogMetadata> newPartitionsToResto
     }
 
     @Override
-    public void remove(final Collection<TopicPartition> revokedChangelogs) {
+    public void unregister(final Collection<TopicPartition> revokedChangelogs) {
```

Comment:
I know this might make the changes a bit harder to follow in the review, but I think this renaming will make things less confusing in the long run to be symmetrical to `register`

Reply:
Thanks! We just have to review this once, but maintain it "forever", so I'm always happy to see improvements like this.